wallet.js

```diff
@@ -198,12 +198,25 @@ async function fetch_most_recent_unconfirmed_send() {
     const IGNORE_UTXOS_BELOW_SATS = get_min_sat_utxo_limit();
 
     if (WALLET_TYPE === 'core') {
-        const recent_transactions = listtransactions()
-        const all_unconfirmed_sends = recent_transactions.filter(
+        const recent_transactions_all_outputs = listtransactions({ count: 200 })
+        const all_unconfirmed_sends = recent_transactions_all_outputs.filter(
             it => it.category === 'send' && it.confirmations === 0
         )
-        const unconfirmed_sends= all_unconfirmed_sends.filter( it => BTC_to_satoshi(it.amount) >= IGNORE_UTXOS_BELOW_SATS)
-        const num_unconfirmed_send_below_limit = all_unconfirmed_sends.length - unconfirmed_sends.length
+        // Sort by most negative (largest send first)
+        const all_unconfirmed_sends_sorted = all_unconfirmed_sends.sort((a, b) => b.amount - a.amount)
+        // For a send with multiple outputs (common in an extraction tx), listtransactions() will return an entry for each output
+        // in the same tx. So we have many entries referring to the same txid, and should just grab one of them.
+        const unique_txids = new Set()
+        const unique_unconfirmed_sends = all_unconfirmed_sends_sorted.filter(it => {
+            if (unique_txids.has(it.txid)) {
+                // This assumes all_unconfirmed_sends_sorted is sorted by biggest (most negative) amounts first
+                return false
+            }
+            unique_txids.add(it.txid)
+            return true
+        })
```

Comment:
Shall this work as expected but with shorter syntax?
```
return unique_txids.has(it.txid) ? false : unique_txids.add(it.txid)
```

![image](https://github.com/deezy-inc/sat-hunter/assets/159653027/daa3cdf7-ee95-4b00-a1b4-4024c7a87d55)

Reply:
hmm so `.add` returns a Set?

Reply:
i'd prefer to just leave it, little more clear

Reply:
although a Set is "true-y" i think its better to return the actual value `true`